cob_hardware_config/cob4-2/config/teleop.yaml

```diff
@@ -2,6 +2,7 @@
 ##########
 # common params
 run_factor: 2.0
+apply_ramp: false
```

Comment:
good point. Adapted to all robots in https://github.com/ipa320/cob_robots/commit/a342522e2c05822f94cc843001bd6ae0f6c2380f